ci-operator/config/kubernetes/descheduler/e2e.json

```diff
@@ -0,0 +1,44 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_rpm_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.9"
+  },
+
+  "canonical_go_repository": "github.com/kubernetes-incubator/descheduler",
+
+  "binary_build_commands": "hack/verify-gofmt.sh && make build",
+  "rpm_build_commands": "make build-rpms",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "descheduler",
+      "context_dir": "images/descheduler/",
+      "dockerfile_path": "Dockerfile.origin"
+    }
+  ],
+
+  "resources": {
+    "*": {
+      "requests": { "cpu": "100m", "memory": "200Mi" },
+      "limits":   { "cpu": "2",    "memory": "4Gi" }
+    }
+  }
```

Comment:
Necessary?

These are copied from the unit tests job, I'm not sure what values are
appropriate.

Reply:
would need to iterate later on these